src/test/java/api/loans/CheckOutByBarcodeTests.java

```diff
@@ -1443,21 +1445,66 @@ void checkOutRefusedWhenAutomatedBlockExistsForPatron() {
   }
 
   @Test
-  void checkOutFailsWhenEventPublishingFailsWithBadRequestError() {
+  void checkOutShouldNotFailIfEventPublishingFailsWithBadRequestError() {
```

Comment:
do we have a test case that returns a 201 status code?

Reply:
Sure, the most of the tests use checkOutFixture.checkOutByBarcode which checks inside 201 status